### PR TITLE
Add logout button UI

### PR DESCRIPTION
**Pull request by simonyandev**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Used the shared Button component to make a styled component for the Log out button, tweaked a part of the UI structure of the header to place the items as needed., and updated the tests to cover the rendering of the Log out button.

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/App.tsx [hunk 1]
```diff
@@ -47,25 +47,31 @@ const App: React.FC = () => {
...
+              </S.Hyperlink>
 
-            <S.NavbarItem>
-              {GIT_TAG && <Version tag={GIT_TAG} commit={GIT_COMMIT} />}
-            </S.NavbarItem>
+              <S.NavbarItem>
+                {GIT_TAG && <Version tag={GIT_TAG} commit={GIT_COMMIT} />}
+              </S.NavbarItem>
+            </S.NavbarBrand>
+            <S.LogoutButton buttonType="primary" buttonSize="M">
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/App.tsx:72]: so this is a button without any functionality ?

Comment (simonyandev) [reply kafka-ui-react-app/src/components/App.tsx:72]: Yes, I've talked with Roma, he said only the UI is required. The backend is not ready yet for the logout action.

Comment (simonyandev) [reply kafka-ui-react-app/src/components/App.tsx:72]: I've mentioned in the PR that this is a UI addition.

**Review by Mgrdich — APPROVED** [review]

**Review by rAzizbekyan — APPROVED** [review]

**Review by rAzizbekyan — APPROVED** [review]

**Review by workshur — APPROVED** [review]
This PR is blocked by BE

**Review by lazzy-panda — APPROVED** [review]

**Comment by Haarolean** [comment]
@rAzizbekyan let's do a redirect to "/logout", that's it

**Pull request merged**